install/amazon_rds/_04_setting_env_vars_multi.mdx

```diff
@@ -0,0 +1,40 @@
+import { APIKeyInstructions } from './04_configure_the_collector_ec2.mdx'
+
+export const CollectorDotEnv = ({ apiKey }) => {
+  return (
+    <CodeBlock>
+      {`CONFIG_CONTENTS="[pganalyze]
+api_key = ${apiKey ?? 'your_pga_organization_api_key'}\n
+[server1]
+db_host = instance-id1.account-id.us-east-1.rds.amazonaws.com
+db_name = your_database_name
+db_username = your_monitoring_user
+db_password = your_monitoring_user_password
+db_sslrootcert = rds-ca-global
+db_sslmode = verify-full\n
```

Comment:
Why do you have these escaped newlines only at the end of some lines?

Reply:
Good question, I kinda copy&pasted from the other place (EC2) but I was also wondering about this inconsistency too. Let me try to remove it and see if it still works.

Reply:
Ok I tested this, but looks like gatsby doesn't like an empty line (causes parse errors), so this workaround is needed.